torchvision/__init__.py

```diff
@@ -105,7 +105,7 @@ def _is_tracing():
     "this issue: https://github.com/pytorch/vision/issues/6753, and you can also "
     "check out https://github.com/pytorch/vision/issues/7319 to learn more about "
     "the APIs that we suspect might involve future changes. "
-    "You can silence this warning by calling torchvision.disable_beta_transform_warning()."
+    "You can silence this warning by calling torchvision.disable_beta_transforms_warning()."
```

Comment:
For reference:

https://github.com/pytorch/vision/blob/beb4bb706b5e13009cb5d5586505c6d2896d184a/torchvision/__init__.py#L112